README.md

```diff
@@ -100,335 +154,353 @@ immunopepper make_bg \
+8. **Annot_IS_SUCCESS**: Empty file indicating that background generation was successful. Generated only if `--skip-annotation` is set to False.
+9. **Output_sample_IS_SUCCESS**: Empty file created if the foreground generation was successful.
+10. **\[mut_mode\]_sample_peptides_meta**: File containing details for each peptide generated from an exon pair.
+
+Detail explanation for columns in **\[mut_mode\]_sample_peptides_meta**
+- **peptide**: str. The peptide sequence for a specific exon pair of the foreground data.
+- **id**: In the format of \[gene_name\]:\[first vertex\]_\[second vertex\]:\[variant_id\]:\[translation_start_position\]:\[kmerType\]. Eg: `ENSG00000198515.13:18_15:0:47952701:2-exons`.
+`ENSG00000198515.13` is the gene name, `0_2` means this junction consists of vertex 0 and vertex 2. `0` means there is no
+somatic mutation or that it is the first case of all somatic mutation combination cases. `47952701 ` is the translation start position. `2-exons` means that the peptide is made by the combination of 2 exons.
+- **readFrame**: int (0,1,2). Reading frame used for translating the peptide.
+- **readFrameAnnotated**: True if the reading frame was appearing in the annotation file provided under `--ann-path`. The flag value will the False if the reading frame was obtained with the immunopepper tool.
+- **geneName**: str. The name of Gene.
+- **geneChr**: str. The Chromosome id where the gene is located.
+- **geneStrand**: str ('+', '_'). The strand of gene.
+- **mutationMode**: str ('ref', 'somatic', 'germline', 'somatic_and_germline'). Mutation mode
+- **hasStopCodon**: int. Indicate if there is stop codon in the junction pair.
+- **isJunctionList**: (np.nan, 1, 0). Indicate if the junction pair appear in the given junction whitelist provided under `--gtex-junction-path`.
+- **isIsolated**: int. Indicate if the output peptide is actually translated from a single exon instead of two.
+- **variantComb**: If mutation files are provided, it shows the somatic mutation combination used in this line of output.
+    eg. 5;25 means the somatic mutation of position 5 and 25 take effect in this output. # TODO: include this or just the example down?
+- **variantSegExpr**: If mutation file and count file are provided, this field shows the expression of segments where the somatic mutation is in.  If mutation files are not provided the value is set to nan.
+    eg. 257.0;123.2 means the segment where the somatic mutation in position 5 is in has counts 257.0 #TODO: include this example?
+- **modifiedExonsCoord**: Coordinates for the exons forming the junction. Usually we have 4 number start_v1;stop_v1;start_v2;stop_v2. It is obtained taking into account the CDS reading frame.
+- **originalExonsCoord**: Shows the original exon coordinates obtained from splice graph without taking into account the CDS.
+- **vertexIdx**: shows the vertex id of the given junction. Eg: 5,6 means this junction pair consists of the fifth and
```

Comment:
Add: Vertex numbering is relative to the splicegraph

Reply:
added an explanation with this